util/test_timeline.py

```diff
@@ -161,36 +162,13 @@ def parse_report(args):
 
     return entries, datetime.fromtimestamp((report_start_time + start_time) / 1000)
 
-
-def parse_time(timestamp):
-    """Parses a timestamp into a datetime object"""
-    return datetime.strptime(timestamp, '%Y-%m-%dT%H:%M:%S')
-
-
-def parse_event_time(event):
```

Comment:
Ah, this is nice cleanup too!

Reply:
Do you think it's worth moving the timestamp definition as well? It is technically duplicated, but I feel a bit weird about having argument definitions defined outside the file.

Reply:
I think it'd be fine to move as they both really do share it.